#get_objects_map

string map
# flag: if need objectsInfo. False: objectsInfo will be empty. 
# when SRSFurnitureGeometry unavailable, corresponding items (e.g. x, y, z) will be set -1000 which is certainly invalid for srs. Or the whole will be set null
# default: true
bool ifGeometryInfo

---

string[] objects
string[] classesOfObjects
string[] spatialRelation       # e.g. aboveOf, under, behind etc
string[] spatialRelatedObject  # e.g. Table0,  Bed0,  Door0, etc   # 
string[] houseHoldId
srs_msgs/SRSSpatialInfo[] objectsInfo
string[] readableNames
